using-ide-vscode.md

```diff
@@ -84,23 +84,25 @@ as a Flutter project. Please ensure the folder that contains your `pubspec.yaml`
 *Note*: If the status bar reads **No Devices** Flutter has not been able to discover any connected
 iOS or Android devices or simulators. You need to connect a device, or start a simulator, to proceed.
 
-The Flutter extension will automatically select the last device connected, however if you have multiple
-devices/simulators connected you can click on the device in the status bar to be presented with a pick-list
-at the top of the screen to allow you to select the device to use for running/debugging.
+The Flutter extension automatically selects the last device connected,
+however if you have multiple devices/simulators connected you can click
+on the device in the status bar to be presented with a pick-list
+at the top of the screen that allows you to select the device to use for
+running/debugging.
 
 ### Run app without breakpoints
 
 1. Click **Debug>Start Without Debugging** in main IDE window or press `Ctrl`+`F5`.
-* The status bar will turn orange to show you are in a debug session
-* The bottom **Debug Console** will show output:<br>
+* The status bar turns orange to show you are in a debug session
```

Comment:
Bullet items should end in periods (https://developers.google.com/style/periods#bulleted-lists).